cfme/tests/v2v/test_v2v_migrations_single_vcenter.py

```diff
@@ -345,17 +344,13 @@ def test_migration_restart(request, appliance, provider,
         target_provider=provider,
         vm_list=mapping_data_vm_obj_single_datastore.vm_list,
     )
-    view = navigate_to(migration_plan, "InProgress")
     assert migration_plan.wait_for_state("Started")
 
     def _system_reboot():
-        # reboot system when migrated percentage greater than 20%
-        ds_percent = int(view.progress_card.get_progress_percent(migration_plan.name)["datastores"])
-        if ds_percent > 10:
-            appliance.restart_evm_rude()
-            return True
-        else:
-            return False
+        # reboot system when the actual disk migration elapsed a 240 second time duration
+        migration_plan.in_progress(plan_elapsed_time=240)
+        appliance.restart_evm_rude()
+        return True
```

Comment:
Wouldn't you need to return False in any condition?

Reply:
No, we do not require any True and False condition here even the wait_for is also not required, I have updated it.